docs/tools/offline/cli-test-debug.md

```diff
@@ -99,6 +143,12 @@ You can specify to only **build** the tests by using the `--compile` option:
 $ mbed test -m K64F -t GCC_ARM --compile
 ```
 
+In case of icetea only the test applications are built
```

Comment:
Again, what do you mean by "In case of Icetea..."?  If you mean regarding or concerning Icetea, then you need 'the':" in [the] case of... ", but this is still not the best way to say this.  "In case" is only use without 'the' when saying something like "In case you were wondering..." (it's followed by a noun/subject).
So, are you saying that: 1) if you use the same compile option as above, only the test applications will build in Icetea? Or, 2) If they want to specify that only the test applications build in Icetea, then they should use the following command? (Is this elaborating on the previous command, or is it its own separate instruction).

Reply:
There are 2 distinct tools, greentea and icetea. Greentea is used by default and to run some commands for icetea you need to use some command line arguments for that.